src/test/java/ibm/jceplus/junit/openjceplus/TestResetByteBuffer.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright IBM Corp. 2024
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution.
+ */
+
+package ibm.jceplus.junit.openjceplus;
```

Comment:
Does this test not apply to the FIPS provider too?

Reply:
This did cross my mind that perhaps we wanted to test this with the OpenJCEPlusFIPS provider in addition to just OpenJCEPlus. The code through both these paths should be identical. This being said better safe then sorry so I turned this test on for OpenJCEPlusFIPS with updates in commit https://github.com/IBM/OpenJCEPlus/commit/64d8350ad66b6459c4c9e3629a90b9cf7194c1db